_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhdp.theme

```diff
@@ -512,4 +512,14 @@ function rhdp_preprocess_assembly(&$variables) {
       $eloqua_assembly_id = $assembly->id();
       $variables['eloqua_assembly_id'] = $eloqua_assembly_id;
     }
+
+    if ($assembly->hasField('field_code')) {
+      $field_code = $assembly->get('field_code')->getValue()[0]['value'];
+	  $variables['code'] = preg_replace(
+		'/(<pre><code>(.)+<\/code><\/pre>)/',
+		'',
```

Comment:
I feel like regexp replacements is always a sure sign of trouble. Why is this bit necessary? Can we accomplish whatever this is after in a different way?